GliaWidgets/Sources/CallVisualizer/Coordinator/CallVisualizer.Coordinator.swift

```diff
@@ -127,20 +127,20 @@ extension CallVisualizer {
             }
         }
 
-        func showConfirmationAlert() {
+        func showConfirmationAlert(_ answer: Command<Bool>) {
```

Comment:
Can we make Coordinator class body smaller to silence the body length warning?

Reply:
Fixed 246e9175ed076c002b5ba21490d0622a73767154